packages/form-core/test/validate/ResultValidator.test.js

```diff
@@ -3,18 +3,29 @@ import { ResultValidator } from '../../src/validate/ResultValidator.js';
 import { Required } from '../../src/validate/validators/Required.js';
 import { MinLength } from '../../src/validate/validators/StringValidators.js';
 
+/**
+ * @typedef {import('../../src/validate/Validator').Validator} Validator
+ */
+
 describe('ResultValidator', () => {
   it('has an "executeOnResults" function returning active state', async () => {
     // This test shows the best practice of creating executeOnResults method
     class MyResultValidator extends ResultValidator {
-      executeOnResults({ regularValidateResult, prevValidationResult }) {
-        const hasSuccess = regularValidateResult.length && !prevValidationResult.length;
-        return hasSuccess;
+      /**
+       *
+       * @param {Object} context
+       * @param {Validator[]} context.regularValidationResult
+       * @param {Validator[]} context.prevValidationResult
+       * @returns {boolean}
+       */
+      executeOnResults({ regularValidationResult, prevValidationResult }) {
+        const hasSuccess = regularValidationResult.length && !prevValidationResult.length;
+        return !!hasSuccess;
       }
     }
     expect(
       new MyResultValidator().executeOnResults({
-        regularValidateResult: [new Required(), new MinLength(3)],
+        regularValidationResult: [new Required(), new MinLength(3)],
```

Comment:
Agree with the name change, but potentially breaking

Reply:
Actually I didn't change the name, I just aligned it with the source code. I got a type error that showed that regularValidateResult prop does not exist, so yay for types showing us we have a bug in our test :). too bad the test still passed before this change though